jetbrains-core/src/software/aws/toolkits/jetbrains/settings/EcsExecCommandSettings.kt

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.settings
+
+import com.intellij.openapi.components.PersistentStateComponent
+import com.intellij.openapi.components.ServiceManager
+import com.intellij.openapi.components.State
+import com.intellij.openapi.components.Storage
+
+@State(name = "ecsExec", storages = [Storage("aws.xml")])
+class EcsExecCommandSettings : PersistentStateComponent<AwsEcsExecConfiguration> {
+    private var state = AwsEcsExecConfiguration()
+
+    override fun getState(): AwsEcsExecConfiguration? = state
+
+    override fun loadState(state: AwsEcsExecConfiguration) {
+        this.state = state
+    }
+
+    var showExecuteCommandWarning: Boolean
+        get() = state.showExecuteCommandWarning
+        set(value) {
+            state.showExecuteCommandWarning = value
+        }
+
+    companion object {
+        @JvmStatic
+        fun getInstance(): EcsExecCommandSettings = ServiceManager.getService(EcsExecCommandSettings::class.java)
```

Comment:
ServieManager is deprecated in IJ master (https://github.com/JetBrains/intellij-community/blob/master/platform/core-api/src/com/intellij/openapi/components/ServiceManager.java)

Use `ApplicationManager.getInstance().getService()`

Reply:
Changed to ApplicationManager.getApplication.getService()